README.md

```diff
@@ -281,14 +282,14 @@ In order to test that each module added to `nf-core/modules` is actually working
 
         ```console
         cd /path/to/git/clone/of/nf-core/modules/
-        TMPDIR=~ PROFILE=singularity pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        TMPDIR=~ PROFILE=singularity pytest --tag star --symlink --wt 2 --keep-workflow-wd
         ```
 
     - Typical command with Conda:
 
         ```console
         cd /path/to/git/clone/of/nf-core/modules/
-        PROFILE=conda pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        PROFILE=conda pytest --tag star --tag star_align_paired_end --symlink --wt 2 --keep-workflow-wd
```

Comment:
We should try and keep the same command for Conda / Docker / Singularity to highlight how to run using those different options. Also need to change the command for Docker.

Reply:
I was trying to highlight the different ways you could use pytest-workflow. But I guess they'll just have to read the docs, not worth the confused people making issues.